src/client/js/WebGME.js

```diff
@@ -113,6 +115,16 @@ define([
                 );
 
                 WebGMEGlobal.State.setIsInitPhase(true);
+                WebGMEGlobal.State.on('change', function (model, options) {
```

Comment:
I would set the document.title = 'WebGME' here and update to a "projectName" on the 'CLIENT_CONSTANTS.PROJECT_OPENED' at line 148 below.
The console.log is still there too..